scripts/tools/fetch-pull-request

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ $# -eq 0 ]
+    then
+        echo "Id of pull request not supplied"
```

Comment:
Can you please make the message as `Pull request id not given.`. Also add a help message like to execute this script give 2 arguments.